### DISCUSSION TITLE
When will 5.8.2 be released for the snowflake provider?

### DISCUSSION THREAD

**Discussion by haarismirza** [apache/airflow#45004] [General]
Looking to utilize a feature that was merged in after the 5.8.1 release

https://github.com/apache/airflow/tree/main/providers/src/airflow/providers/snowflake

**Comment by potiuk** [comment]
We release providers every two weeks more-or-less. And you can always buid locally your own provider (see https://github.com/apache/airflow/blob/main/dev/breeze/doc/09_release_management_tasks.rst#preparing-provider-packages) or copy the source code with changes to have your own operator/hook if you want to make it faster.
